content/en/tutorials/vpc_peering.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "AWS VPC Peering"
+linkTitle: "AWS VPC Peering"
+date: 2021-07-21
+draft: false
+description: Guide on how to peer AWS VPCs
+---
+
+## What is VPC Peering?
+VPC peering is the standard way of opening _private_ network communication between 2 AWS VPCs. This means a server in
+one VPC can send requests to a private IP (e.g. another server, a database, etc...) in the other network, without
+touching the public internet. This ability is extremely useful if one wishes to connect different Opta environments
+(or a pre-existing VPC and an Opta environment) together without any serious security risk. This guide will show
+you step-by-step instructions on how to do it.
+
+## Limitation?
+VPCs can peer from cross AWS account and even cross region with minimal difficulties, but they can never peer
+with another VPC that shares an overlapping CIDR block.
+
+Furthermore, AWS security groups do not work across VPC peerings-- you will need to whitelist by individual IPs or CIDR
+blocks.
```

Comment:
there is also more known limitations, but not sure if they are relevant to us https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-basics.html#vpc-peering-limitations